templates/repo/graph/commits.tmpl

```diff
@@ -0,0 +1,80 @@
+                        {{if eq $refGroup "pull"}}
+                            {{if $.HidePRRefs}}
+                                {{if (containGeneric $.SelectedBranches .Name) }}
+                                    <a class="ui labelled icon button basic tiny" href="{{$.RepoLink}}/pulls/{{.ShortName|PathEscape}}">
+                                        {{svg "octicon-git-pull-request" 16 "mr-2"}}#{{.ShortName}}
+                                    </a>
+                                {{end}}
+                            {{else}}
+                                <a class="ui labelled icon button basic tiny" href="{{$.RepoLink}}/pulls/{{.ShortName|PathEscape}}">
+                                    {{svg "octicon-git-pull-request" 16 "mr-2"}}#{{.ShortName}}
+                                </a>
+                            {{end}}
+                        {{else if eq $refGroup "tags"}}
+                            <a class="ui labelled icon button basic tiny" href="{{$.RepoLink}}/src/tag/{{.ShortName|PathEscape}}">
+                                {{svg "octicon-tag" 16 "mr-2"}}{{.ShortName}}
+                            </a>
+                        {{else if eq $refGroup "remotes"}}
+                            <a class="ui labelled icon button basic tiny" href="{{$.RepoLink}}/src/commit/{{$commit.Rev}}">
+                                {{svg "octicon-cross-reference" 16 "mr-2"}}{{.ShortName}}
+                            </a>
+                        {{else if eq $refGroup "heads"}}
+                            <a class="ui labelled icon button basic tiny" href="{{$.RepoLink}}/src/branch/{{.ShortName|PathEscape}}">
+                                {{svg "octicon-git-branch" 16 "mr-2"}}{{.ShortName}}
+                            </a>
+                        {{else}}
+                            <!-- Unknown ref type {{.Name}} -->
+                        {{end}}
+                    {{end}}
+                    —
+                    <span>{{RenderCommitMessage $commit.Subject $.RepoLink $.Repository.ComposeMetas}}</span>
+                    —
+                    <span class="author">
+                        {{$userName := $commit.Commit.Author.Name}}
+                        {{if $commit.User}}
+                            {{if $commit.User.FullName}}
+                                {{$userName = $commit.User.FullName}}
+                            {{end}}
+                            <img class="ui avatar image" src="{{$commit.User.RelAvatarLink}}" alt=""/><a href="{{AppSubUrl}}/{{$commit.User.Name}}">{{$userName}}</a>
+                        {{else}}
+                            <img class="ui avatar image" src="{{AvatarLink $commit.Commit.Author.Email}}" alt=""/>{{$userName}}
+                        {{end}}
+                    </span>
+                    <span class="time">{{$commit.Date}}</span>
+                {{ end }}
+            </li>
+        {{ end }}
+    </ul>
+</div>
```

Comment:
This and other new files seem to be space-indented, can you tab-indent please? (Is editorconfig working?)

Reply:
I don't think it is!